types/babar/babar-tests.ts

```diff
@@ -0,0 +1,35 @@
+import babar from "babar";
```

Comment:
The `import babar = require("babar")` form is preferred for CommonJS modules.

Reply:
@amcasey, I know this already merged. But im curious. Why is this form preferred? I am still learning this stuff.

Reply:
There are more thorough write-ups in the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md) and the [handbook](https://www.typescriptlang.org/docs/handbook/modules.html), but here's the _totally unofficial_ way I think about it - there are fundamentally two kinds of modules:
 1) CommonJS modules, which use `module.exports =` and `const x = require`
 2) ES6 modules, which use `export x` and `import x from`

There are a bunch of flags that make them kind of work together, but I generally try to avoid mixing them.

Now, you've probably noticed that `import x = require` wasn't in either list - that's because that's TS-specific.  `export =` and `import x = require` are constructs that TS understands (unlike `const x = require`, which is just a function call), so it can "do the right thing".  So, since babar is a CommonJS module, the appropriate import is `const babar = require("babar")`, as indicated in their samples.  But an _even better_ import is `import babar = require("babar")`, because TS understands that form and can do smart things for you.

Hope that helps.

Reply:
Yeah! Thank you.